### ISSUE TITLE
Black image result

### ISSUE THREAD

**Issue by manu15sd** [Hans1984/SelfTMO#1]
I'm getting a black image as a result for the training. This occurs when I adjust the parameters, such as the input size or the number of training epochs, but it also occasionally occurs during regular training when nothing is changed. I'm utilizing my own 32-bit per channel HDRs for the training.

**Comment by Hans1984** [comment]
Hi, thanks for the attention. It's hard to tell what happen there from the limited information. However, do you check the intensities of the HDR images you use. If you use .exr images, from my experience, sometimes it will appear nagative values after do resizing.

**Comment by manu15sd** [comment]
I'm using.exr; however, I've noticed that sometimes an identical image yields a black output, while other times it yields a standard one, so I think that it is not about intensities because some times the result is correct.

**Comment by Hans1984** [comment]
Ok, I haven't encountered such problem. My suggestion is that to use a fixed seed for hyper parameters initialization.

**Comment by manu15sd** [comment]
I can't find an exposed seed anywhere

**Comment by Hans1984** [comment]
I mean to set a fixed seed, something like, torch.manual_seed(seed_value)  in  pytorch (you could check that in TF).

**Comment by manu15sd** [comment]
Yep, thank you, that worked.
